models/electric_relaxation/src/utils/set_paths.py

```diff
@@ -14,7 +14,7 @@ def get_artifacts_path(partition_name):
 
 def get_data_path(data_name):
     '''
-    E.g., The data is saved in data/raw/raw.parquet
+    E.g., The data is saved in data/raw/raw_calibration.parquet
     '''
 
     return Path(__file__).parent.parent.parent / "data" / f"{data_name}" / f"{data_name}.parquet"
```

Comment:
Can I get you to adopt this solution instead:

PATH = Path(__file__)
sys.path.insert(0, str(Path(*[i for i in PATH.parts[:PATH.parts.index("views_pipeline")+1]]) / "common_utils")) 
from set_path import setup_project_paths, setup_data_paths
setup_project_paths(PATH)

PATH_RAW, PATH_PROCESSED, PATH_GENERATED = setup_data_paths(PATH)

Reply:
Same above

Reply:
Super - yes, I also saw you had the "right" solution elsewhere